### PR TITLE
Create Network upgrade postmortem template.md

### PR DESCRIPTION
**Pull request by poojaranjan**
Network upgrade postmortem report template.

When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

**Comment by MicahZoltu** [comment]
Same comments as over in #2809, I don't think the EIP repository is the right place for post mortems of Ethereum mainnet operations.

**Comment by poojaranjan** [comment]
This PR is created to add a template to create the network upgrade postmortem report and be saved as an Informational EIP for community reference. 

Being a template, I've added Type = Meta (Ref. EIP 233, a template for creating the hardfork meta EIP). However, this is not a proposal related to the network standard, so, if suggested, "Type' can be changed to "Informational EIP" to get it merged. 

IMO, changing the type of proposal will let this template sit as a recommendation and not a mandate to be followed to create the network upgrade postmortem/assessment report.

File: EIPS/Network upgrade postmortem template.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+---
+eip: TBD
+title: Recommendation for creating Ethereum network upgrade analysis/postmortem report
+author: Pooja Ranjan 
+discussions-to: 
+status: Draft
+type: Meta
```
Comment (lightclient) [inline EIPS/Network upgrade postmortem template.md:7]: I think this should be informational.

Comment (lightclient) [reply EIPS/Network upgrade postmortem template.md:7]: Sorry, I take that back. I think *this* EIP should be meta, but it should specify that EIPs that follow this template are *informational*.

**Pull request closed without merging**